fcos-policy-engine/src/utils.rs

```diff
@@ -0,0 +1,36 @@
+use actix::prelude::*;
+use commons::graph;
+use failure::{Error, Fallible};
+use reqwest::Method;
+use std::time::Duration;
+
+/// Default timeout for HTTP requests (30 minutes).
+const DEFAULT_HTTP_REQ_TIMEOUT: Duration = Duration::from_secs(30 * 60);
+/// Default address of fcos-graph-builder, which is the same as fcos-policy-builer
+const DEFAULT_GB_ADDR: &str = "http://127.0.0.1:5050/v1/graph";
```

Comment:
Not sure if this is appropriate :(

Reply:
Yes, this is fine (it will eventually be moved to a configuration entry).